src/api/app/helpers/webui/request_helper.rb

```diff
@@ -7,13 +7,26 @@ module Webui::RequestHelper
     'superseded' => 'red'
   }.freeze
 
+  STATE_BOOTSTRAP_ICONS = {
+    'new' => 'fa-code-branch',
+    'review' => 'fa-search',
+    'accepted' => 'fa-check',
+    'declined' => 'fa-hand-paper',
+    'revoked' => 'fa-eraser',
+    'superseded' => 'fa-plus'
+  }.freeze
+
   AVAILABLE_TYPES = ['all', 'submit', 'delete', 'add_role', 'change_devel', 'maintenance_incident', 'maintenance_release'].freeze
   AVAILABLE_STATES = ['new or review', 'new', 'review', 'accepted', 'declined', 'revoked', 'superseded'].freeze
 
   def request_state_color(state)
     STATE_COLORS[state.to_s] || ''
   end
 
+  def request_bootstrap_icon(state)
+    STATE_BOOTSTRAP_ICONS[state.to_s] || ''
```

Comment:
would it make any different if this return `nil`? or in other words, why don't write it without the `|| ''`

Reply:
I don't know. I just used what was there already and trusted that this was done for a reason.